eras/allegra/impl/CHANGELOG.md

```diff
@@ -7,6 +7,10 @@
   * `PState`
   * `VState`
 
+## 1.1.0.1
+
+* Fix an issue where `TotalDeposits` didn't appear on Allegra and Mary era
```

Comment:
We have not released 1.1.0.0 yet, so you should add your changelog entry there. We recently documented our new release process: https://github.com/input-output-hk/cardano-ledger/blob/e15e3545814140c63a972b1938372a075c288f6a/RELEASING.md#changelogmd

Reply:
I was wondering if this bugfix could be backported to the existing release branches for the 3 packages it affects, as mentioned at https://github.com/input-output-hk/cardano-ledger/blob/e15e3545814140c63a972b1938372a075c288f6a/RELEASING.md#release-process?

Reply:
How much major? Should it bump the first or the second release number? I bumbed the second for now.

Reply:
According to PVP version `A.B.C.D`:
* D - patch: non-semantic changes
* C - minor version: non-breaking changes
* B - major version: breaking changes
* A - complete rewrite :slightly_smiling_face:

Addition of a constructor to a type is a breaking change, so `B` needs a bump